### PR TITLE
Add active committers API implementation

### PR DESCRIPTION
**Pull request by ganeshkumarsv**
New Method for fetching active committers for an Organization
https://docs.github.com/en/rest/reference/billing#get-github-advanced-security-active-committers-for-an-organization

### CODE REVIEW

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:118]: I'm realizing now that I did not do a very thorough job of reviewing #1797, and therefore this `billing.go` does not match the style for the rest of this repo.

I'd like to fix that, and maybe you can assist me.  If you could please apply these modifications to all the endpoints within `billing.go` (and the third change to all the endpoints in `billing_test.go`), that would be greatly appreciated... and please do not use `force-push` if possible to make the reviews much easier.

So first, after line 117, please add a blank line.

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:120]: Second... after every `s.client.Do`, please add this code (including the trailing blank line):

```
if err != nil {
  return nil, resp, err
}

```

File: github/billing_test.go
Comment (gmlewis) [inline github/billing_test.go:436]: Third, please add (see `users_test.go` or other files for more examples):

```

	testNewRequestAndDoFailure(t, methodName, client, func() (*Response, error) {
		got, resp, err := client.Billing.GetAdvancedSecurityActiveCommittersOrg(ctx, "o")
		if got != nil {
			t.Errorf("testNewRequestAndDoFailure %v = %#v, want nil", methodName, got)
		}
		return resp, err
	})
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:53]: I think `Repositories` is too general and can be easily confused with `Repository`.

```suggestion
// RepositoryActiveCommitters represents active committers on each repository.
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:50]:
```suggestion
	Repositories                    []*Repositories `json:"repositories,omitempty"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:57]:
```suggestion
	AdvancedSecurityCommittersBreakdown []*AdvancedSecurityCommittersBreakdown `json:"advanced_security_committers_breakdown,omitempty"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:60]:
```suggestion
// AdvancedSecurityCommittersBreakdown represents the user activity breakdown for ActiveCommitters.
```

**Review by gmlewis — CHANGES_REQUESTED** [review]

**Comment by ganeshkumarsv** [comment]
@gmlewis I guess I addressed all the review comments. Please let me know if everything looks alright. Thanks!

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:49]: If it isn't a reference type (e.g. `*int` in this case) then it doesn't need the `omitempty`.

```suggestion
	TotalAdvancedSecurityCommitters int                          `json:"total_advanced_security_committers"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:55]: These fields might not get populated, so let's make them reference types (and leave the `omitempty`):

```suggestion
	Name                                *string                                `json:"name,omitempty"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:56]:
```suggestion
	AdvancedSecurityCommitters          *int                                   `json:"advanced_security_committers,omitempty"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:57]:
```suggestion
	AdvancedSecurityCommittersBreakdown []*AdvancedSecurityCommittersBreakdown `json:"advanced_security_committers_breakdown,omitempty"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:62]:
```suggestion
	UserLogin      *string `json:"user_login,omitempty"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:63]:
```suggestion
	LastPushedDate *string `json:"last_pushed_date,omitempty"`
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:135]: `ActiveOrgCommitters` should not be capitalized.

**Review by gmlewis — CHANGES_REQUESTED** [review]

File: github/billing.go
Comment (ganeshkumarsv) [reply github/billing.go]: @gmlewis can I know how we find out if its a reference type or not?

**Comment by ganeshkumarsv** [comment]
@gmlewis I guess I addressed all the comments. Please let me know if the changes look alright! Thanks for the review!

File: github/billing.go
Comment (gmlewis) [reply github/billing.go]: > @gmlewis can I know how we find out if its a reference type or not?

There are a few scenarios where reference types are needed and/or desired:

* When the response might not populate the field,
* When an option being sent from the client to the server is optional and we don't want the `zero` value of the field to be sent
* When we have a slice of a struct, it is desirable to iterate over a slice of pointers rather than a slice of values.

Conversely, if a field is already a reference type (for example, a slice or an interface), we don't typically want to have a pointer to it.  In other words, if you ever see `*[]something` or *[]*something` then that is typically suspect and not desirable. 

I hope that helps.

**Review by gmlewis — APPROVED** [review]
Thank you, @ganeshkumarsv !
LGTM.

Awaiting second LGTM before merging.

(Please note that ALL other contributors to this repo are welcome to provide the second PR review/comment/approval that we need for merging and that we are not waiting for any particular reviewer unless otherwise noted.)

**Comment by ganeshkumarsv** [comment]
@cpanato @gunadhya @sagar23sj 
I see you have contributed to `billing.go` and `billing_test.go`. Can anyone please review and approve my PR? 🙇

**Review by cpanato — APPROVED** [review]
looks great to me, thanks for adding that

**Comment by gmlewis** [comment]
Thank you, @cpanato !
Merging.

**Comment by ganeshkumarsv** [comment]
@gmlewis can we have a minor release?

**Comment by gmlewis** [comment]
> @gmlewis can we have a minor release?

I will work on a release, but due to the breaking API change in #2205, it will be a major release.

**Comment by gmlewis** [comment]
@ganeshkumarsv - this change is now incorporated in the new release:
https://github.com/google/go-github/releases/tag/v41.0.0

**Pull request merged**
